src/main/java/com/redfin/sitemapgenerator/SitemapGenerator.java

```diff
@@ -163,6 +165,38 @@ THIS getThis() {
 		return outFiles;
 	}
 	
+	/**
+	 * Writes out the sitemaps as strings

```

Comment:
It might surprise a user that this returns multiple strings. We should clarify why it returns a list, and what will be in it.